tensorflow/python/ops/nn_grad.py

```diff
@@ -390,6 +390,21 @@ def _Relu6GradGrad(op, grad):
           array_ops.zeros(shape=array_ops.shape(x), dtype=x.dtype))
 
 
+@ops.RegisterGradient("LeakyRelu")
+def _LeakyReluGrad(op, grad):
+  x = op.inputs[0]
+  alpha = op.get_attr("alpha")
+  return gen_nn_ops.leaky_relu_grad(grad, x, alpha=alpha)
+
+
+@ops.RegisterGradient("LeakyReluGrad")
+def _LeakyReluGradGrad(op, grad):
```

Comment:
Can you add a gradient checker test for these? Use https://www.tensorflow.org/api_docs/python/tf/test/compute_gradient_error

Reply:
Well, I believe that python UTs for GradGrad was already in the original commit.